### ISSUE TITLE
how  about the model size?

### ISSUE THREAD

**Issue by dagongji10** [divamgupta/image-segmentation-keras#239]
I have trained the model with `model_name=fcn_8_vgg`. And the checkpoint file is 1.6G.
Is that normal?
![image](https://user-images.githubusercontent.com/15357846/92571926-514a1280-f2b6-11ea-848f-a570efe6fd03.png)

**Comment by divamgupta** [comment]
Yes

**Comment by dagongji10** [comment]
@divamgupta 
When do you usually stop training?To what extent is the general accuracy considered good enough for the model?
I got acc 0.7941, and class_wise_iou：[0.5007, 0.8113, 0.4269 , 0.6884, 0.6243, 0.4782, 0.6933, 0.5241]. If it's on a generic data set, isn't that usually a bad result?

**Comment by divamgupta** [comment]
Ideally you should have a separate validation set, and you train until the validation accuracy increases.

**Comment by varungupta31** [comment]
Hey @dagongji10 
I'm trying to train this model on custom data as well, but am runninng into some issues.
May I know which version of tensorflow you are using.
If you are using tf<2.4.1, help me out with the changes you made in the code to get it running.

Thanks.
